NOTICE.txt

```diff
@@ -19,7 +19,7 @@ http://rafael.xavier.blog.br
 https://github.com/rxaviers/cldrjs/blob/master/LICENSE-MIT
```

Comment:
@DokaRus 

This link is broken - https://github.com/rxaviers/cldrjs/blob/master/LICENSE-MIT

Let's fix it in context of this PR?

Reply:
Sure